Samples/Client/SwiftClient/SwiftClient/VideoStreamViewController.swift

```diff
@@ -0,0 +1,640 @@
+        var urlRequest = URLRequest(url: urlString)
+        urlRequest.setValue("Client", forHTTPHeaderField: "Peer-Type")
+        hangingGet = URLSession.shared.dataTask(with: urlRequest, completionHandler: hangingGetCallback)
+        hangingGet.resume()
+    }
+    
+    func startHeartBeat() {
+        if !heartBeatTimerIsRunning {
+            DispatchQueue.main.async {
+                self.heartBeatTimer = Timer.scheduledTimer(timeInterval: TimeInterval(self.heartbeatIntervalInSecs), target: self, selector: #selector(VideoStreamViewController.startHeartBeat), userInfo: nil, repeats: true)
+            }
+            heartBeatTimerIsRunning = true
+        }
+        let url = URL(string: "\(server)/heartbeat?peer_id=\(myId!)")!
+        var urlRequest = URLRequest(url: url)
+        urlRequest.setValue("Client", forHTTPHeaderField: "Peer-Type")
+        let heartbeatGet = URLSession.shared.dataTask(with: urlRequest)
+        heartbeatGet.resume()
+    }
+    
+    func signInCallback(data: Data?, response: URLResponse?, error: Error?) {
+        let response = response as! HTTPURLResponse
+        if response.statusCode == 200 && error == nil {
+            if let data = data, let stringData = String(data: data, encoding: String.Encoding.utf8) {
+                let peers = stringData.components(separatedBy: "\n")
+                myId = Int(peers[0].components(separatedBy: ",")[1])!
+                print("My id: \(myId)")
+                
+                for (index, peer) in peers.enumerated() {
+                    if index >= 1 && peer.characters.count > 0 {
+                        print("Peer \(index): \(peer)")
+                        let parsed = peer.components(separatedBy: ",")
+                        otherPeers[Int(parsed[1])!] = parsed[0]
+                    }
+                }
+                startHangingGet()
+                startHeartBeat()
+                updatePeerList()
+                
+                request = nil
+            }
+        }
+    }
+    
+    func signIn() {
+        let urlString = URL(string: "\(server)/sign_in?peer_name=\(localName)")!
+        var urlRequest = URLRequest(url: urlString)
+        urlRequest.setValue("Client", forHTTPHeaderField: "Peer-Type")
+        request = URLSession.shared.dataTask(with: urlRequest, completionHandler: signInCallback)
+        request.resume()
+    }
+    
+    func sendToPeer(peerId: Int, data: Data) {
+        print(peerId, " Send ", data)
+        if myId == -1 {
+            print("Not connected")
+            return
+        }
+        if peerId == myId {
+            print("Can't send a message to oneself :)")
+            return
+        }
+        
+        let urlString = URL(string: "\(server)/message?peer_id=\(myId!)&to=\(peerId)")!
+        var urlRequest = URLRequest(url: urlString)
+        urlRequest.httpMethod = "POST"
+        urlRequest.httpBody = data
+        urlRequest.setValue("Client", forHTTPHeaderField: "Peer-Type")
```

Comment:
again, not needed, probably should remove

Reply:
I will remove all Peer-Type related code.